getblocklab/block-lab
# Add more component tests for controls

#### Changes
* Adds component tests for all controls that didn't have them
* Continues #514
* Should give us even more confidence in each commit. Also, we should be able to merge a lot of `dependabot` PRs without any manual testing.

#### Testing instructions
1. `npm install`
1.  `npm run test:js`
1. Expected: there are only dots, no error messages
1. Verify that `classic-text`, `range`, and `select` controls still work (they have minor edits here)
